### DISCUSSION TITLE
Using Magicbell/headless crashing in  react native.

### DISCUSSION THREAD

**Discussion by rawalhussainkhan** [magicbell/community#149] [Questions] [answered]
I'm trying to use magicbell/headless in my react native for push notifications according to the documentation but getting this crash can anyone have a solution for this or anyone else is facing this. 

**React Native Version:0.67.3** 
**React Version:17.0.2**
![Screenshot 2023-11-14 at 2 20 07 PM](https://github.com/magicbell-io/community/assets/134264831/ae979991-c86f-4081-ab43-47b6834527ab)
**

**Comment by unamashana** [comment, marked as answer]
Hey @rawalhussainkhan very sorry about missing this. Please let us know if you still need help.
